packages/core/src/test/__fixtures__/entities.ts

```diff
@@ -350,6 +351,131 @@ export const entryWithEmbeddedAssetInRichText: Entry = {
   },
 };
 
+export const entryWithDeeplyEmbeddedEntitiesInRichText: Entry = {
+  sys: {
+    id: entityIds.ENTRY_WITH_DEEPLY_EMBEDDED_ENTITIES_IN_RICH_TEXT,
+    type: 'Entry',
+    contentType: {
+      sys: {
+        id: 'bar',
+        type: 'Link',
+        linkType: 'ContentType',
+      },
+    },
+    createdAt: '2020-01-01T00:00:00.000Z',
+    updatedAt: '2020-01-01T00:00:00.000Z',
+    revision: 10,
+    space: {
+      sys: {
+        type: 'Link',
+        linkType: 'Space',
+        id: 'cfexampleapi',
+      },
+    },
+    environment: {
+      sys: {
+        id: 'master',
+        type: 'Link',
+        linkType: 'Environment',
+      },
+    },
+    locale: 'en-US',
+  },
+  fields: {
+    body: {
+      nodeType: 'document',
+      data: {},
+      content: [
+        {
+          nodeType: 'unordered-list',
+          data: {},
+          content: [
+            {
+              nodeType: 'list-item',
+              data: {},
+              content: [
+                {
+                  nodeType: 'embedded-asset-block',
+                  data: {
+                    target: {
+                      sys: {
+                        type: 'Link',
+                        id: entityIds.ASSET1,
+                        linkType: 'Asset',
+                      },
+                    },
+                  },
+                  content: [],
+                },
+              ],
+            },
+            {
+              nodeType: 'list-item',
+              data: {},
+              content: [
+                {
+                  nodeType: 'embedded-entry-block',
+                  data: {
+                    target: {
+                      sys: {
+                        type: 'Link',
+                        id: entityIds.ENTRY1,
+                        linkType: 'Entry',
+                      },
+                    },
+                  },
+                  content: [],
+                },
+              ],
+            },
+            {
+              nodeType: 'list-item',
+              data: {},
+              content: [
+                {
+                  nodeType: 'asset-hyperlink',
+                  data: {
+                    target: {
+                      sys: {
+                        id: entityIds.ASSET1,
```

Comment:
Should this have a different ID like `ASSET2` so it's not confused with the "embedded-asset-block"?